Composer/packages/server/src/utility/creation.ts

```diff
@@ -4,21 +4,36 @@
 import { BotTemplate, QnABotTemplateId } from '@bfc/shared';
 import union from 'lodash/union';
 
-export const templateSortOrder = [
+import AssetService from '../services/asset';
+
+export const backUpTemplateSortOrder = [
   { generatorName: '@microsoft/generator-bot-empty', displayName: 'Blank bot' },
-  { generatorName: '@microsoft/generator-bot-conversational-core', displayName: 'Basic conversational bot' },
+  { generatorName: '@microsoft/generator-bot-conversational-core', displayName: 'Basic bot' },
   { generatorName: '@microsoft/generator-bot-assistant-core', displayName: 'Basic assistant' },
   { generatorName: '@microsoft/generator-bot-enterprise-assistant', displayName: 'Enterprise assistant' },
   { generatorName: '@microsoft/generator-bot-people', displayName: 'People' },
   { generatorName: '@microsoft/generator-bot-calendar', displayName: 'Calendar' },
   { generatorName: QnABotTemplateId, displayName: 'QnAMaker bot' },
 ];
 
-export const sortTemplates = (templates: BotTemplate[]): BotTemplate[] => {
+export const sortTemplates = async (templates: BotTemplate[]): Promise<BotTemplate[]> => {
```

Comment:
this is not sorting templates, it's fetching an array of data with a certain order.

Reply:
Its passed an unsorted list of templates, uses the sort order which is fetched from its own external source, then uses that sort order to build and return a sorted list of templates.

Reply:
my bad, it was hiding the unchanged part. :)